src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        action="index.html"
```

Comment:
The `action` attribute of the form is set to 'index.html'. This will cause the form to reload the same page upon submission. If this is not the intended behavior, consider changing the `action` attribute to point to a different URL or script that handles form submissions.